NEWS.rst

```diff
@@ -9,6 +9,82 @@
 
 .. towncrier release notes start
 
+21.1 (2021-04-24)21.1 (2021-04-24)
+=================
```

Comment:
This looks wrong. :/

Reply:
@sbidoul I'm looking into how to fix this w/ towncrier. Seems like this is related to the NEWS handling changes done recently.

https://meet.google.com/fuw-tjbw-xyc is where I'll be sitting if you wanna join. :)

Reply:
Hm, I'm going to fix that manually.

Reply:
Sounds good! You'll need to amend the git tag too, if you're doing that.

Reply:
@pradyunsg I wanted to say hi, but you meet link above is incomplete.

Reply:
> [meet.google.com/fuw-tjbw-xyc](https://meet.google.com/fuw-tjbw-xyc) is where I'll be sitting if you wanna join. :)

I've left.

Reply:
@pradyunsg looks like this broken related to that towncrier compat "fix" https://github.com/pypa/pip/pull/9817.

I've found out that `towncrier` produces a broken title while `towncrier --draft` (that my sphinx extension uses) produces a good one.

This problem has manifested itself in setuptools too: https://github.com/pypa/setuptools/issues/2666. I'll probably be investigating this soonish and track the progress there.